### PR TITLE
update pd init flags

### PR DESCRIPTION
**Pull request by v01dstar**
PTAL @shenli @iamxy

### CODE REVIEW

File: pd.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   type: NodePort
   ports:
-  - name: etcd-server
+  - name: etcd-client
```
Comment (shenli) [inline pd.yaml]: We don't have etcd now. Why here is etcd-client?

File: tidb.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ spec:
         image: pingcap/tidb:latest
         ports:
         - containerPort: 4000
-        args: ["-L=info", "--store=tikv", "--path=pd:2379/pd?cluster=1", "-P=4000"]
+        args: ["-L=info", "--store=tikv", "--path=pd:2379?cluster=1", "-P=4000"]
```
Comment (shenli) [inline tidb.yaml]: There is a flag named "--status". We need to set it.

**Comment by shenli** [comment]
PTAL @iamxy

**Comment by shenli** [comment]
@tangfx PTAL

File: pd.yaml (hunk 1 above)
Comment (siddontang) [inline pd.yaml]: use client instead

Btw, do we need to set peer ?

Comment (v01dstar) [inline pd.yaml]: This service is created to serve those outcoming requests like TIDB or TIKV, so it only needs to expose 1234, 2379, 9090.
While the 3 services bellow are created to help a single node to find its peers. So they expose 2380.

**Comment by iamxy** [comment]
LGTM

**Pull request merged**